vizro-core/src/vizro/models/_components/card.py

```diff
@@ -39,6 +39,7 @@ def build(self):
             dbc.NavLink(
                 children=text,
                 href=get_relative_path(self.href) if self.href.startswith("/") else self.href,
+                target="_blank",
```

Comment:
I'm curious how come internal links like `/page-2` at the end (UI client side) end up without `target="_blank"` specified, and the external link ends up with the `target="_blank"`? It looks like we are adding `target="_blank"` here for both external and internal links. 🤔

<img width="753" alt="image" src="https://github.com/user-attachments/assets/89d3d40d-3ea3-466a-a2fc-d9217d91bf0a">

Reply:
This is von the docs linked in the PR description: 

> target (string; optional): Target attribute to pass on to the link. Only applies to external links.

Reply:
`dbc.NavLink` only adds `target` for external links - see https://dash-bootstrap-components.opensource.faculty.ai/docs/components/nav/. Note that the detection of whether something is `external_link` itself is found automatically on the dbc side, which is very useful for us.

I was also curious how this worked so found the relevant bit of code yesterday 😀 https://github.com/facultyai/dash-bootstrap-components/blob/c9367e6284823df7118594c5f55e8caf488c6a6d/src/private/Link.js#L84

So basically this is a great bit of design by dbc and makes our lives easy 😀